app/js/test.js

```diff
@@ -0,0 +1,20 @@
+var submit = document.querySelector('#btn')
+var url = 'http://localhost:8080/test'
+
+postData(url, submit)
+
+async function postData(url, submit) {
+    submit.addEventListener('click', async function (e) {
+        e.preventDefault()
+        var testName = document.querySelector('#testName').value
```

Comment:
suggest changing id to test_id to be consistent with other code and database structure